src/shared/logger/activation.ts

```diff
@@ -25,40 +25,99 @@ const DEFAULT_LOG_LEVEL: LogLevel = 'info'
 /**
  * Activate Logger functionality for the extension.
  */
-export async function activate(extensionContext: vscode.ExtensionContext): Promise<void> {
-    const outputChannel = LOG_OUTPUT_CHANNEL
+export async function activate(
+    extensionContext: vscode.ExtensionContext,
+    outputChannel: vscode.OutputChannel
+): Promise<void> {
+    const logOutputChannel = LOG_OUTPUT_CHANNEL
     const logPath = LOG_PATH
-    const logLevel = getLogLevel()
 
     await fs.ensureDir(path.dirname(logPath))
-    setLogger(makeLogger(logLevel, logPath, outputChannel, extensionContext.subscriptions))
+
+    // default logger
+    setLogger(
+        makeLogger(
+            {
+                logPaths: [logPath],
+                outputChannels: [logOutputChannel],
+            },
+            extensionContext.subscriptions
+        )
+    )
+    getLogger().info(`log level: ${getLogLevel()}`)
```

Comment:
since we are changing this, I suggest (in a separate commit) changing this to `error()` so that it always appears. This only happens once, and it is useful to see explicitly in the log file what the log-level is.